app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```

Comment:
Why not put all adapter (list and view in utils) in there?

Reply:
I think this indeed should be done.